decidim-proposals/app/commands/decidim/proposals/publish_proposal.rb

```diff
@@ -24,6 +24,7 @@ def call
 
         transaction do
           @proposal.update published_at: Time.current
+          Decidim::Gamification.increment_score(@current_user, :proposals)
```

Comment:
Maybe increment the score for all the authors?

Reply:
Will look into that.